calendar_resource/models/calendar_event.py

```diff
@@ -219,36 +229,36 @@ def _check_resource_ids_working_times(self):
 
                     if not intervals and record.allday:
                         conflict_intervals.append(
-                            (datetime_start, datetime_end)
+                            (datetime_start, datetime_end),
                         )
                     else:
                         available_intervals += intervals
 
                 if not record.allday:
                     conflict_intervals = self.env['resource.calendar'].\
```

Comment:
This `\` is killing me. Let's assign the model to an intermediary variable or something instead